doc/mgr/dashboard.rst

```diff
@@ -690,7 +690,11 @@ We provide a set of CLI commands to manage user accounts:
 
 - *Create User*::
 
-  $ ceph dashboard ac-user-create [--force-password] <username> [<password>] [<rolename>] [<name>] [<email>] [--enabled] [<pwd_expiration_date>]
+  $ ceph dashboard ac-user-create [--force-password] <username> [<password>] [<rolename>] [<name>] [<email>] [--enabled] [<pwd_expiration_date>] [--pwd_update_required]
+
+  To bypass the password policy checks use the `force-password` option.
+  Use the option `pwd_update_required` so that the newly created user has
+  to change his password after the first login.
```

Comment:
Small textual improvement suggestion here (using the [singular they](https://www.merriam-webster.com/words-at-play/singular-nonbinary-they)):
```
 Use the option `pwd_update_required` so that a newly created user has
 to change their password after the first login.
```